### PR TITLE
jenkins: no freebsd10 for Node 11

### PR DESCRIPTION
**Pull request by targos**
Refs: https://github.com/nodejs/node/pull/22617

This needs to wait until the PR on nodejs/node is landed.

### CODE REVIEW

File: jenkins/scripts/VersionSelectorScript.groovy [hunk 1]
```diff
@@ -65,6 +65,9 @@ def buildExclusions = [
   // OSX ---------------------------------------------------
   [ /^osx1010/,                       anyType,     gte(11) ],
   [ /^osx1011/,                       releaseType, lt(11)  ],
+  
+  // FreeBSD -----------------------------------------------
+  [ /^freebsd10/,                     anyType,     lt(11)  ],
```
Comment (targos) [inline jenkins/scripts/VersionSelectorScript.groovy]: Wait, I think I did it the opposite way. It should be `gte(11)`, right?

Comment (richardlau) [reply jenkins/scripts/VersionSelectorScript.groovy]: Yes, this is a list of exclusions, see:
https://github.com/nodejs/build/blob/d121160b7d38cecd98c794a2a9549b89708a4754/jenkins/scripts/VersionSelectorScript.groovy#L8-L9

**Comment by targos** [comment]
I just landed https://github.com/nodejs/node/pull/22617.

**Comment by refack** [comment]
[Test job](https://ci.nodejs.org/job/test-node-test-commit-freebsd/5/) looks good.

**Pull request merged**